libs/components/theme/src/lib/theming/theme-brand.ts

```diff
@@ -17,13 +17,16 @@ export class SkyThemeBrand {
   /**
    * Creates a new theme brand.
    * @param name The name of the theme brand.
+   * @param version The version of the theme brand.
    * @param hostClass The class on the host element which child components should reference when
-   * adjusting for a specified theme brand. This defaults to `sky-theme-brand-<name>
+   * adjusting for a specified theme brand. This defaults to `sky-theme-brand-<name>`
+   * @param sriHash Optional subresource integrity hash for the theme stylesheet. cspell:disable-line
```

Comment:
The `cspell:disable-line` part shouldn't be here since this parameter will show in the consumer's IDE as Intellisense. If there's a word that CSpell doesn't recognize then it should go in the CSpell config file.